src/Hng.Web/appsettings.json

```diff
@@ -19,10 +19,10 @@
     "PublicKey": ""
   },
   "SmtpCredentials": {
-    "Host": "",
-    "Port": "",
-    "Username": "",
-    "Password": ""
+    "Host": "host",
+    "Port": "1234",
+    "Username": "user",
+    "Password": "pass"
```

Comment:
Please remove theses dummy values for these secretes

Reply:
I thought the successresponse and errorresponse would automatically do that.